default.nix

```diff
@@ -4,6 +4,7 @@ with pkgs;
 
 let
   inherit (rust.packages.nightly) rustPlatform;
+  darwin = pkgs.callPackage ./darwin { };
```

Comment:
https://github.com/holochain/holochain-rust/blob/85b70e9d9f015198f60ec1c5f075312c69b6c7a0/default.nix#L18-L21 might suffice.

Reply:
I'm going to implement that right now, will take a few minutes. Sorry for the delay.

Reply:
Yes; are you proposing we integrate this into the holo-nixpkgs `buildRustPackage`?  Because, every rust build on a Mac needs this.  In fact, we should just go ahead and include all 3: frameworks.Security frameworks.CoreFoundation frameworks.CoreServices